### ISSUE TITLE
Have you tuned the vision model in sft stage?

### ISSUE THREAD

**Issue by xmy0916** [X-PLUG/mPLUG-Owl#179]
I found code here: https://github.com/X-PLUG/mPLUG-Owl/blob/main/mPLUG-Owl2/scripts/finetune.sh#L32

you have freezed the vision backbone, but according to your paper, it seems all the parrams have been trained?
![image](https://github.com/X-PLUG/mPLUG-Owl/assets/43675899/e9f5782b-768c-4c05-b18a-7bade7d588a0)

**Comment by MAGAer13** [comment]
Yes, this is just reimplementation of our model. You can enable it during training and you should implement layer-wise learning rate by yourself.

**Comment by xmy0916** [comment]
> Yes, this is just reimplementation of our model. You can enable it during training and you should implement layer-wise learning rate by yourself.

Thanks for your great job and answer~
